index.html

```diff
@@ -40,6 +40,222 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <span>A daily selection of privately</span>
+                        <span>personalized reads;no accounts or</span>
+                        <span> sign-ups required.</span>
+                    </div>
+                    <div class="primarytext3">
+                        <button class="lang">HTML</button>
+                        <button class="lang">CSS</button>
+                        <button class="lang JavaScript">JavaScript</button>
+                    </div>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="works-container">
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio(5).png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <div class="multi-post">
+    
+                            Multi-Post
+                            <br>
+                            Stories
+    
+                        </div>
+    
+                        <div class="fram">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <span>A daily selection of privately</span>
+                        <span>personalized reads;no accounts or</span>
+                        <span> sign-ups required.</span>
+                    </div>
+                    <div class="primarytext3">
+                        <button class="lang">HTML</button>
+                        <button class="lang">CSS</button>
+                        <button class="lang JavaScript">JavaScript</button>
+                    </div>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="about-my-slef">
+                <div class="top">
+                    <div class="first-text">About <br> Myself</div>
+                    <div class="second-text">About Myself
```

Comment:
- [ ] Please ensure that this paragraph is wrapped in a `p` tag to improve your website accessibility because the `p` tag semantically represents the descriptions of a section 👍🏽